### PR TITLE
Version 0.2

### PR DESCRIPTION
**Pull request by AHReccese**
#### Reference Issues/PRs
Version 0.2 Release PR

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -85,6 +85,7 @@ body:
       description: Which version of Reserver are you using?
       options:
         - Reserver 0.1
+        - Reserver 0.2
```
Comment (sadrasabouri) [inline .github/ISSUE_TEMPLATE/bug_report.yml:88]: Put the latest version at the top. Take a look at [Nava's bug_report.yml](https://github.com/openscilab/nava/blob/main/.github/ISSUE_TEMPLATE/bug_report.yml?plain=1) file as an example.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,11 +6,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+### Changed
+## [0.2] - 2024-06-17
+### Added
+- `CLI` Handler
```
Comment (sadrasabouri) [inline CHANGELOG.md:12]:
```
- `CLI` handler
```

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Thanks Amirhossein. Minor changes are needed here.

**Review by sepandhaghighi — APPROVED** [review]
🚀

File: CHANGELOG.md (hunk 2 above)
Comment (AHReccese) [reply CHANGELOG.md]: Applied.

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**